Formula/fceux.rb

```diff
@@ -3,7 +3,7 @@ class Fceux < Formula
   homepage "https://fceux.com/"
   url "https://github.com/TASEmulators/fceux.git",
       tag:      "fceux-2.6.0",
-      revision: "c075043727920bdd1250184c2b6bd33585c7ab43"
+      revision: "1ca8b8e9d0c401b8a9687b019af30aa088d1c4d2"
```

Comment:
Why was this retagged? Do we need to publish new bottles or bump the revision?

Reply:
The upstream repo retagged it due to something that was missed. So a hot fix was applied and re tagged. I was surprised that homebrew jumped on it so quickly

Reply:
What was missed exactly? That determines whether we have to ship new builds and/or bump the revision (which will tell users who had the older `2.6.0` that their installation is outdated).

Reply:
By the way, retagging your releases is generally not a nice thing to do to your downstream users. The [Git docs](https://git-scm.com/docs/git-tag#_on_re_tagging) have some rather strong words about retagging.

Reply:
A small code patch. The application needs to be recompiled

Reply:
The release wasn't even out for a full 24 hours and the patch was only a few lines. Next time I will stage it for a few days.

Reply:
It doesn't really matter how long a tag has been up: switching out your tags is likely to silently break someone's clone of your repository.